prompts/2024/english/prompt_2024_01/AnaColon_summary_Bingham_and_Ratcliff_2024

```diff
@@ -0,0 +1,17 @@
+Complex multicellularity has evolved five separate times in eukaryotes but has never evolved in prokaryotes. Complex multicellularity has been defined as a large organism composed of many cells that lives for an extensive period of time. Some prokaryotes are multicellular but they do not have the morphologies of complex multicellular organisms. 
+
+This paper proposes a new hypothesis as to why prokaryotes lack complex multicellularity compared to eukaryotes. When bottle neck events causing a reduced efficient population size cells are divided into groups. In response to this eukaryotes will undergo genomic expansion while prokaryotes will undergo genomic deletion. These differences in responses could be an explanation as to why eukaryotes have been able to evolve complex multicellularity and prokaryotes have not. 
+
+Eukaryotic have transposons and introns that facilitate genomic expansion through replication. Prokaryotes tend to have deletion bias where they will delete parts of their genome. These subcellular mechanisms influence their response to bottleneck effects. 
+
+With an expanded genome, eukaryotes have more potential for the emergence of new genes. This would facilitate their evolution after experiencing a bottleneck effect with a reduced population size. Prokaryotes would lose genetic material which would cause less genetic diversity as they try to evolve. Less genomic material leads to less opportunity for evolutionary events to occur. 
```

Comment:
An important point in the paper is that, perhaps because of these large genomes (or maybe enabling the large genomes), eukaryotes tend to have 'default off' gene expression. So when these random duplications and fragmentations happen, they accumulate as 'junk DNA' that's not automatically 'used'. This makes the copying much less dangerous.